e2e/tests-dfx/call.bash

```diff
@@ -39,6 +39,13 @@ teardown() {
   assert_eq '(record { c = "A"; d = "B" })'
 }
 
+@test "call without argument, using candid assistant" {
+  install_asset echo
+  dfx_start
+  dfx deploy
+  assert_command "${BATS_TEST_DIRNAME}/../assets/expect_scripts/candid_assist.exp"
+}
+
```

Comment:
Can we have one of these for `deploy` and `install` as well?

Here is the output I see when deploying to a canister declared as `actor class c(p: { x: Nat; y: Int;}) `:
```
Installing canisters...
Installing code for canister dd_backend, with canister ID bkyz2-fmaaa-aaaaa-qaaaq-cai
Enter field x : nat
  ✔ Enter a nat · 6
Enter field y : int
  ✔ Enter an int · 8
Sending the following argument:
(record { x = 6 : nat; y = 8 : int })

Do you want to send this message? [y/N]
```


Suggested wording:

```
Installing canisters...
Installing code for canister dd_backend, with canister ID bkyz2-fmaaa-aaaaa-qaaaq-cai

Canister dd_backend requires an initialization argument.
Enter field x : nat
  ✔ Enter a nat · 6
Enter field y : int
  ✔ Enter an int · 8

Initialization argument for dd_backend will be:
(record { x = 6 : nat; y = 8 : int })

Do you want to initialize canister dd_backend with this argument? [y/N]
```

Reply:
Done

Reply:
Here's what I see now, when deploying a canister with declaration `actor class c(x: Principal) {`

```
Installing code for canister dd_backend, with canister ID bkyz2-fmaaa-aaaaa-qaaaq-cai
This canister requires the following initialization argument.
Auto-completions: alice, anonymous, bob, dd_backend, dd_frontend, default, mainnet
? Enter a principal › 
```

Note "This canister requires the following initialization argument." but no following description of the argument.  The term "requires the following initialization argument" would have to refer to a "following" description of that argument.  It can't be used to refer only to the upcoming prompt for that argument.

Reply:
Changed back to "This canister requires an initialization argument."